package.json

```diff
@@ -12,17 +12,17 @@
   "author": "",
   "license": "ISC",
   "engines": {
-    "node": "^14.0.0"
+    "node": ">=14.0.0"
```

Comment:
@hassanabral could you please provide some context why was it limited to v14? I assume the idea was to prevent using a lower version

Reply:
@aleks-gentux I limit it to node v14 to make sure it's compatible with node version in other repositories. I tried to upgrade `gentux-eslint-config` in `shared-ecom-js` to `v3.0.2` and I'm getting node version incompatible error :(

```
➜  shared-ecomm-js (master) yarn                                                                                                                                                                                                                  ✭ ✱
yarn install v1.22.19
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
warning Resolution field "eslint-plugin-react@7.33.0" is incompatible with requested version "eslint-plugin-react@^7.33.2"
[3/5] 🚚  Fetching packages...
error @typescript-eslint/eslint-plugin@6.17.0: The engine "node" is incompatible with this module. Expected version "^16.0.0 || >=18.0.0". Got "14.19.3"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Reply:
@hassanabral I see... I didn't get that error cause I'm updating shared-ecomm-js to use node 20